webpack/assets/javascripts/react_app/components/common/EmptyState/EmptyState.test.js

```diff
@@ -36,60 +22,7 @@ describe('Default Empty State', () => {
 });
 
 const emptyStatePatternFixtures = {
-  'icon, header and description are mandatory': {
-    icon: 'printer',
-    header: 'Printers',
-    description: 'Printers print a file from the computer',
-  },
-  'should render description when given one as a text': {
-    icon: 'printer',
-    header: 'Printers',
-    description: 'Printers print a file from the computer',
-    documentation: 'This is a simple description',
-  },
-  'should render description when given one as JSX': {
-    icon: 'printer',
-    header: 'Printers',
-    description: 'Printers print a file from the computer',
-    documentation: (
-      <div>
-        This is my <b>description</b>.
-      </div>
-    ),
-  },
-  'should render main action when given title and url': {
-    icon: 'printer',
-    header: 'printers',
-    description: 'printers print a file from the computer',
-    action: (
-      <PrimaryActionButton
-        action={{ title: 'my title', url: 'https://somewhere.com' }}
-      />
-    ),
-  },
-  'should render main action when given title and onClick': {
-    icon: 'printers',
-    header: 'printers',
-    description: 'printers print a file from the computer',
-    action: (
-      <PrimaryActionButton
-        action={{ title: 'my title again', onClick: () => '' }}
-      />
-    ),
-  },
-  'should render secondary action when given one': {
-    icon: 'printer',
-    header: 'printers',
-    description: 'printers print a file from the computer',
-    secondaryActions: (
-      <SecondaryActionButtons
-        actions={[
-          { title: 'x', url: 'some-url' },
-          { title: 'y', url: 'random-url' },
-        ]}
-      />
-    ),
-  },
+  'should render with props': props,
```

Comment:
I liked that you reduced the amount of this snapshots tests, looking at the props it seem they had equivalent props and almost the same snapshots :+1: